pocs/utils/social_twitter.py

```diff
@@ -0,0 +1,30 @@
+import tweepy
+
+from pocs.utils.logger import get_root_logger
+
+
+class SocialTwitter(object):
+
+    """Messaging class to output to Twitter
+    """
+    logger = get_root_logger()
+
+    def __init__(self, consumer_key, consumer_secret, access_token, access_token_secret, output_timestamp):
```

Comment:
I wonder whether this class should just get a dict (from the yaml reader) representing the config inside the twitter block of the config file. That would allow the caller to be relatively independent of the set of configs needed by one of these classes.

Reply:
That is certainly possible, however the style of coding in the other modules I saw is to handle the config in the 'main' module rather than the utils module. I think that is how PanMessaging works for example. 

I thought passing the parameters rather than a dictionary was a little clearer as it's a sort of DI of the stuff this class needs

Reply:
OK I'll look into this thanks